doc/package-blueprint.yaml

```diff
@@ -0,0 +1,72 @@
+# Other properties than designer are not consolidated.
+# They are just plain as they are in Mashery
+package:
+  name: Some package
+  description: Package description
+  eav: {}
+  keyLength: 24
```

Comment:
Generate Shared Secrets='true or enabled'
Shared Secret Length=10

Reply:
I see. These properties are after "plans" when you download detail of package and I missed them. Add with another commit.